src/api.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios'
+
+const newsApi = axios.create({baseURL: 'https://ncnewshostserver.onrender.com/api'})
+
+const getArticles = ()=>{
+    return newsApi
+    .get("/articles")
+    .then((response) => {
+        return response.data;
+    });
+}
+
+const getArticleById = (articleId)=>{
+    return newsApi
+    .get(`/articles/${articleId}`)
+    .then((response)=>{
+        return response.data
+    })
```

Comment:
You could also add a catch block here to console.log any errors that are returned from your axios request